### ISSUE TITLE
'Unread' text in Messages inbox fails WAVE test

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#360]
The color contrast for the red (#FF5959) 'Unread' text against the white background fails a WAVE test. 

Perhaps we could change it to #333333 to match  the 'Delete' text?

Thanks!

**Comment by boonebgorges** [comment]
I've made the change.

**Comment by bree-z** [comment]
Thanks Boone! This looks good.
